tests/content.test.ts

```diff
@@ -22,8 +22,12 @@ describe(`MDC Formatter`, async () => {
 
       const formatted = mdcFormatter(content, { tabSize: 2 })
 
+      // Expect the formatted `input` content to be the same as the expected `output` content
+      expect(formatted).toBe(expected)
```

Comment:
No change here, just moving the assertion for the formatted input/output above the assertion logic for errors.